omod-1.8/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_8/UserResource1_8.java

```diff
@@ -137,19 +140,22 @@ public DelegatingResourceDescription getCreatableProperties() {
 	public UserAndPassword1_8 newDelegate() {
 		return new UserAndPassword1_8();
 	}
-	
+
 	/**
 	 * @see org.openmrs.module.webservices.rest.web.resource.impl.DelegatingCrudResource#save(java.lang.Object)
 	 */
 	@Override
 	public UserAndPassword1_8 save(UserAndPassword1_8 user) {
-		User openmrsUser = new User();
-		String password = user.getPassword();
-		openmrsUser = Context.getUserService().saveUser(user.getUser(), password);
-		return new UserAndPassword1_8(openmrsUser);
-		
-	}
-	
+        User openmrsUser = new User();
+        String password = user.getPassword();
+        openmrsUser = Context.getUserService().saveUser(user.getUser(), password);

```

Comment:
I think we need some smarter logic around here otherwise this won't work well, see my comment on the ticket